import_export/resources.py

```diff
@@ -424,9 +424,11 @@ def import_field(self, field, instance, row, is_m2m=False, **kwargs):
 
     def get_import_fields(self):
         import_fields = []
+        missing = object()
```

Comment:
What is the benefit of using `object` here (rather than `None`)?

Reply:
Something can be an entry, but map to `None`, I want to avoid cases where the check then fails, although I think for most it indeed does not matter.